app/serializers/application.js

```diff
@@ -19,6 +19,10 @@ export default HalSerializer.extend({
   normalize: function(type, hash, property) {
     var payload = this._super(type, hash, property);
 
+    if(payload["default"]) {
+      payload["isDefault"] = payload["default"];
```

Comment:
Oof, this seems quirky. To make a single point, only data coming from the server will be changed to `isDefault`. When a record is saved, `isDefault` will not be mapped back to `default`.

I suggest using `attrs` on a serializer:

http://emberjs.com/api/data/classes/DS.RESTSerializer.html#property_attrs

This would allow you to be very specific here. In `app/serializers/vhost.js`:

```js
// ...
  attrs: {
    isDefault: 'default'
  }
```

This way only vhost has the mapping.